### ISSUE TITLE
base: Confusing term "Company" in contact form

### ISSUE THREAD

**Issue by pedrobaeza** [odoo/odoo#93552]
**Impacted versions:** all, but know that this can only be changed in master

**Steps to reproduce:**

- Go to Contacts
- Create a new record
- You see this screen

![Selección_005](https://user-images.githubusercontent.com/7165771/173522205-c5c91fbd-c852-4bd5-aca8-6ec6144d33d5.png)

**Current behavior:**

The term "Company" confuses users, mixed with the field "Company" (`company_id`) that is the core of the multi-company feature of Odoo.

**Expected behavior:**

Use another term to refer to this. Maybe "Corporation" or "Enterprise".

**Comment by pedrobaeza** [comment]
@mart-e @fhe-odoo I would like to have your opinion on this.

**Comment by mart-e** [comment]
Could you elaborate the "confuses users" ? I don't find this very confusing, it's a radio button (as opposed to m2o field). I don't see how it's confused with the multi-company feature.

**Comment by bouvyd** [comment]
I'm not sure it would be less confusing to move to a different naming, this is a rather well-established field - the cascade renaming we would need to put in place to be coherent is rather huge and the ROI is honestly not huge from my PoV

if we were to change, to stay coherent, i suppose we would set the radio checkboxes to Individual/Organisation, then rename `commercial_company_id` to *Organisation* and `company_name` *Organisation Name* and *company_type* to `entity_type` (or whatever) - and the list could get longer

I can understand why you'd suggest this change, but the cost to do it comprehensively is rather high with no guarantee of a concrete improvement (and risk of further confusion).

**Comment by fhe-odoo** [comment]
I agree with @bouvyd. To me, the fact that it is a radio button next to another one called "individual" gives enough context to not confuse this with a company m2o.

**Comment by pedrobaeza** [comment]
Thanks all for the feedback. I can perform all the needed changes in a PR to master if we agree that this should be tackled. This has been initiated due to a confusion in my users about seeing the term "Company" / "Companies" used in a lot of places with different meaning, and they have really the doubt about what each of them means.

Some examples:

![Selección_005](https://user-images.githubusercontent.com/7165771/173665264-d09c28c1-80d8-4408-a498-f7af84b5f78b.png)

![Selección_006](https://user-images.githubusercontent.com/7165771/173665294-9dc03181-70b8-4294-b8fb-b8ae56c9f935.png)

![Selección_007](https://user-images.githubusercontent.com/7165771/173665326-d49d3673-0cfc-43d6-8efd-894c3dd6dac9.png)

Don't you think this deserves the effort to homogenize the terminology across the entire ERP?

**Comment by karlggest** [comment]
Hi.

IMHO users can be confused with this, and it will be nice avoid estrange explications.

**Comment by bodedra** [comment]
Plenty will be more if we are planning to change "company" selection database value.

+1 for label changing to Organization.

**Comment by davidtranhp** [comment]
I prefer Organization, that opposites Individual. Company/Enterprise may not be ok for the cases that the partner is not a company. For example it is a state authority/entity.

**Comment by gdgellatly** [comment]
It is worse than this. Partners is just the worst for grouping and filtering in multicompany, you never know what the company field means. Every time I think I know and I get the wrong choice out of 3 or 4 similar names.

**Comment by pedrobaeza** [comment]
Well, you can see here in Twitter that the feel is shared by a lot of people. What do you need to reconsider this?

**Comment by bodedra** [comment]
> As stated, I'm not for it. I don't see a clear benefit, the confusion you speak of does not seem to be a globally shared sentiment and change for change's sake is not our style.
> 
> Thank you for your opinions and contributions, as always.

Probably you are thinking of it technically. But in a business process, each term matters a lot. I vote for reconsideration.
